### ISSUE TITLE
Principal Component Analysis code error

### ISSUE THREAD

**Issue by vishnugunneri** [koalaverse/hands-on-machine-learning-with-r#58]
The following code:isn't running-
# How many PCs required to explain at least 75% of total variability
min(which(ve$CVE >= 0.75))
## [1] 27

Might be an issue with variable 've' because there is none
![Hands on ML error](https://user-images.githubusercontent.com/65291056/165578022-ebdb5a52-8d57-4f32-a595-9bf317f7ba9e.png)

**Comment by bradleyboehmke** [comment]
Good catch @vishnugunneri , there is a hidden code chunk that computes `ve` via:

```r
ve <- data.frame(
  PC  = my_pca@model$importance %>% names(),
  PVE = my_pca@model$importance %>% .[2,] %>% unlist(),
  CVE = my_pca@model$importance %>% .[3,] %>% unlist()
)
```
